Test/AbstractWidgetTestCase.php

```diff
@@ -50,36 +53,24 @@ protected function setUp()
         }
         parent::setUp();
 
-        // NEXT_MAJOR: Remove BC hack when dropping symfony 2.4 support
-        $csrfProviderClasses = array_filter([
-            // symfony <=2.4
-            'Symfony\Component\Security\Csrf\CsrfTokenManagerInterface',
-            // symfony >=2.4
-            'Symfony\Component\Form\Extension\Csrf\CsrfProvider\CsrfProviderInterface',
-        ], 'interface_exists');
-
         // TODO: remove the condition when dropping symfony/twig-bundle < 3.2
         if (method_exists('Symfony\Bridge\Twig\AppVariable', 'getToken')) {
             $this->extension = new FormExtension();
             $environment = $this->getEnvironment();
-            $this->renderer = new TwigRenderer(
+            $this->renderer = new FormRenderer(
                 $this->getRenderingEngine($environment),
-                $this->createMock(current($csrfProviderClasses))
+                $this->createMock(CsrfTokenManagerInterface::class)
             );
-            $runtimeLoader = $this
-                ->getMockBuilder('Twig_RuntimeLoaderInterface')
-                ->getMock();
-
-            $runtimeLoader->expects($this->any())
-                ->method('load')
-                ->with($this->equalTo('Symfony\Bridge\Twig\Form\TwigRenderer'))
-                ->will($this->returnValue($this->renderer));
+            $runtimeLoader = new \Twig_FactoryRuntimeLoader([
```

Comment:
You can use this: https://github.com/twigphp/Twig/blob/1.x/src/RuntimeLoader/FactoryRuntimeLoader.php

Twig 1.34 is required.

Reply:
Let's do it later.